### PR TITLE
Specify remote endpoint while seeding config

### PR DESCRIPTION
**Pull request by webengineer**
Consider remote endpoint options. Assume config file is the last argument.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -42,18 +42,30 @@ The most minimalistic viable git2consul configuration mirrors a single git repo
...
 ```
 
-Put that configuration in a file called `/tmp/git2consul.json`.  From the git2consul directory, upload that JSON file into the KV as your git2consul config:
+Put that configuration in a file called `/tmp/git2consul.json`.  From the git2consul directory, upload that JSON file into the local KV as your git2consul config:
 
 ```
 node utils/config_seeder.js /tmp/git2consul.json
 ```
 
+or for remote Consul endopoint:
```
Comment (ryanbreen) [inline readme.md]: spelling on 'endopoint'

File: readme.md [hunk 2]
```diff
@@ -42,18 +42,30 @@ The most minimalistic viable git2consul configuration mirrors a single git repo
...
+node utils/config_seeder.js --endpoint remote.consul.host --port 80 /tmp/git2consul.json
+```
+
 Start git2consul:
 
 ```
 node .
 ```
 
+or for remote Consul endopoint:
```
Comment (ryanbreen) [inline readme.md]: spelling on 'endopoint'

**Comment by ryanbreen** [comment]
This looks really good, and thanks for the contribution!  We have an embarrassment of riches between this patch and GH-39, so we'll have to figure out how to reconcile them. :+1:

**Comment by ryanbreen** [comment]
Thanks!

**Comment by webengineer** [comment]
Thank you too!

**Pull request merged**